docs/learn/learn-auction.md

```diff
@@ -31,63 +31,69 @@ extinguished and the candle went out, the auction would suddenly terminate and t
 that point would win.
 
 When candle auctions are used online, they require a random number to decide the moment of
-termination. Parachain slot auctions differ slightly from a normal candle auction in that it does not 
```

Comment:
Rephrased this sentence which is factually incorrect, as a random number is never used to determine the opening phase - "Parachain slot auctions differ slightly from a normal candle auction in that it does not use the random number to decide the duration of its opening phase." 

to - "Parachain slot auctions differ slightly from a normal candle auction in that they do
not randomly terminate the auction. Instead, they run for an entire fixed duration and the winner is
chosen retroactively."
"